spec/factories/qutes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :quote do
+    description { "some text" }
+    author_name {  Faker::Name.name  }
+    order { Random.rand(1..6)}
```

Comment:
This will fail pretty fast because it'll randomly generate quotes with the same order value. See https://stackoverflow.com/questions/38516923/how-to-sequence-correctly-with-factory-girl-in-rails-development-environment on how to create a sequence.